build.sh

```diff
@@ -118,6 +118,17 @@ write_archive_info() {
     prepare_git_artifacts "${srcdir}" /cosa/coreos-assembler-git.tar.gz /cosa/coreos-assembler-git.json
 }
 
+libguestfish_wrapper(){
```

Comment:
This is totally fine as is...but we could probably just stick this file literally into the source code as e.g. `src/libguestfs-ppc64-wrapper.sh`, have it get installed by the build system as `/usr/lib/coreos-assembler/libguestfs-ppc64-wrapper.sh`.